pkg/storaged/dialog.jsx

```diff
@@ -563,6 +564,27 @@ export const TextInput = (tag, title, options) => {
     };
 };
 
+const PassInputElement = ({ tag, title, options, val, change, validated }) => {
+    const [show, setShow] = useState(false);
+
+    return (
+        <InputGroup>
+            <TextInputPF4 data-field={tag} data-field-type="text-input"
+                          validated={validated}
+                          disabled={options.disabled}
+                          aria-label={title}
+                          autoComplete={options.new_password ? "new-password" : null}
+                          type={show ? "text" : "password"}
+                          value={val}
+                          onChange={(_event, value) => change(value)} />
+            <Button variant="control"
+                    onClick={() => setShow(!show)}
+                    isDisabled={options.disabled}>
+                { show ? <EyeSlashIcon /> : <EyeIcon /> }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18906-20230616-104507-5b7c7f1b-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)